### DISCUSSION TITLE
[HELP]: Exclude games from tiling? If possible.

### DISCUSSION THREAD

**Discussion by Lunasciel** [prasanthrangan/hyprdots#1091] [Q&A]
### Description

Is there a way to exclude games from tiling to make them a normal full screen window like in Windows or Fedora, which does not follow the mouse or jump to another screen after changing the window when it is in focus (or not)?

**Comment by kRHYME7** [comment]
https://wiki.hyprland.org/Configuring/Window-Rules/

https://www.reddit.com/r/hyprland/

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b2561e65-62f9-470f-903c-e7884041dfc8)


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f1de5169-bc77-4f76-8aa1-7d1347f39e1e)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/56139025-15c8-4d2f-8efb-d0b57c121713)



> [!Important]
> Maybe like this ?

``` dwindle:no_gaps_when_only = 1 ```

Preview: 

When only 1 window is open. 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/adf60b88-4b44-4207-8cb2-2bc46256e2d0)


When 2 or more  windows they tiled.  

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/d50342ef-94b5-40ed-a2e1-117beaef0c15)


> [!note]
> almost all games requests a fullscreen.

**Comment by Lunasciel** [comment]
Does this then have to be in a separate config or is it also possible as a launch option in Steam?

**Comment by kRHYME7** [comment]
> Is there a way to exclude games from tiling to make them a normal full screen window like in Windows or Fedora,

Yes? 

> which does not follow the mouse or jump to another screen after changing the window when it is in focus (or not)?

Can you elaborate more? 



If you want to do it exclusively to steam( window specific ), read the [window rules ](https://wiki.hyprland.org/Configuring/Window-Rules/)section on the hyprland wiki. 

Then maybe ask which part you might not get. 

Also try to ask on [reddit](https://www.reddit.com/r/hyprland/) too, as they surely have answers for this.

**Comment by mahaveergurjar** [comment]
ALT + ENTER  key is already for fullscreen like this type see this video

https://github.com/prasanthrangan/hyprdots/assets/57872034/8df29f16-34ef-4751-b7fe-341031fc13dd

**Comment by mahaveergurjar** [comment]
And use of ALT + TAB to switch in windows in full screen

https://github.com/prasanthrangan/hyprdots/assets/57872034/62a08cc4-d3b6-4071-9dac-adc6de77edda

**Comment by kRHYME7** [comment]
Yow I'll be moving this to the discussion panel to segregate issues. Also, to utilize the comment/reply feature.  
Thanks!
